lib/tests/streamlit/runtime/caching/hashing_test.py

```diff
@@ -291,6 +291,23 @@ class EnumClass(Enum):
 
         assert get_hash(enum_1)
 
+    def test_different_enums(self):
```

Comment:
Needs a short docstring describing the intention of the test, e.g.: "Different enum classes should have different hashes, even when the enum values are the same"